pennylane/qchem/convert.py

```diff
@@ -565,18 +565,26 @@ def _ucisd_state(cisd_solver, tol=1e-15):
 def import_state(solver, tol=1e-15):
     r"""Convert an external wavefunction to a state vector.
 
-    Currently, the only accepted source of wavefunctions is the PySCF library.
-    The restricted and unrestricted CISD/CCSD methods are supported.
+    Currently, the only accepted sources of wavefunctions are
+
+        * the PySCF library (the restricted and unrestricted CISD/CCSD
+          methods are supported),
+        * the library Dice implementing the SHCI method,
+        * the library Block2 implementing the DMRG method.
 
     Args:
-        solver: external wavefunction object (e.g. PySCF Solver object) that will be converted
+        solver: external wavefunction object.
+
+            * For PySCF, this is the R/UCISD or R/UCCSD Solver object,
+            * For SHCI via Dice, this is the tuple(list[str], array[float]) of Slater determinants and their coefficients;
+            * For DMRG via Block2, this is the tuple(list[int], array[float]) of Slater determinants and their coefficients
```

Comment:
Not sure if it looks good in the built docs. Please consider merging this information with the bullet points above.

Reply:
Fixed.